src/extensionList.ts

```diff
@@ -0,0 +1,2 @@
+//Imports all extensions for node bot.
+["extension-installer"];
```

Comment:
Since the extension list is a typescript file (probably should be a js file in the config directory, but disregarding that for a moment), it would be better to export and import the file rather than using reading it manually. 

e.g.

```javascript
export const extensions = [
    'extension-installer'
];
```